### ISSUE TITLE
GE2E loss can work with cnn 

### ISSUE THREAD

**Issue by wrongbattery** [HarryVolek/PyTorch_Speaker_Verification#51]
I try to replace your LSTM with CNN but the loss doesn't decrease. The input, loss function are the same. Can you explain why is happening?
